tck/src/main/java/org/eclipse/microprofile/openapi/reader/MyOASModelReaderImpl.java

```diff
@@ -389,6 +400,16 @@ public OpenAPI buildModel() {
                                                                                 .title("id")
                                                                                 .description("id of the new booking")
                                                                                 .addType(
-                                                                                        Schema.SchemaType.STRING)))))))));
+                                                                                        Schema.SchemaType.STRING))))))))
+                        .addPathItem("/refpath/{id}", OASFactory.createPathItem()
+                                .ref("idCrud") // Note PathItem allows ref with other properties
```

Comment:
It's explicit in that `$ref` is listed as a fixed field in a Path Item Object at all and it's not stated that it's mutually exclusive with other fields.

This is in contrast with every other object which does not define a `$ref` field. Instead, at the point where the object may be used, the spec will state that it can either be an object of that type, or a reference object. See for example most of the [fixed fields of a Components object](https://spec.openapis.org/oas/v3.1.0.html#fixed-fields-5).

Our Model API doesn't mirror the spec in this regard - we do have `getRef` and `setRef` on each object which can be referenced, rather than a separate type for references.